packages/grafana-ui/src/components/DateTimePickers/TimeRangePicker/TimePickerFooter.tsx

```diff
@@ -43,42 +53,85 @@ export const TimePickerFooter: FC<Props> = (props) => {
-            includeInternal={true}
-            onChange={(timeZone) => {
-              onToggleChangeTz();
-
-              if (isString(timeZone)) {
-                onChangeTimeZone(timeZone);
-              }
-            }}
-            autoFocus={true}
-            onBlur={onToggleChangeTz}
-          />
-        </section>
-      </div>
-    );
-  }
-
   return (
-    <section aria-label="Time zone selection" className={style.container}>
-      <div className={style.timeZoneContainer}>
-        <div className={style.timeZone}>
-          <TimeZoneTitle title={info.name} />
-          <div className={style.spacer} />
-          <TimeZoneDescription info={info} />
+    <div>
+      <section aria-label="Time zone selection" className={style.container}>
+        <div className={style.timeZoneContainer}>
+          <div className={style.timeZone}>
+            <TimeZoneTitle title={info.name} />
+            <div className={style.spacer} />
+            <TimeZoneDescription info={info} />
+          </div>
+          <TimeZoneOffset timeZone={timeZone} timestamp={timestamp} />
         </div>
-        <TimeZoneOffset timeZone={timeZone} timestamp={timestamp} />
-      </div>
-      <div className={style.spacer} />
-      <Button variant="secondary" onClick={onToggleChangeTz} size="sm">
-        Change time zone
-      </Button>
-    </section>
+        <div className={style.spacer} />
+        <Button variant="secondary" onClick={onToggleChangeTimeSettings} size="sm">
+          Change time settings
+        </Button>
+      </section>
+      {isEditing ? (
+        <div className={style.editContainer}>
+          <div>
+            <RadioButtonGroup
+              value={editMode}
+              options={[
+                { label: 'Time Zone', value: 'tz' },
+                { label: 'Fiscal year', value: 'fy' },
+              ]}
+              onChange={setEditMode}
+            ></RadioButtonGroup>
+          </div>
+          {editMode === 'tz' ? (
+            <section
+              aria-label={selectors.components.TimeZonePicker.container}
+              className={cx(style.timeZoneContainer, style.timeSettingContainer)}
+            >
+              <TimeZonePicker
+                includeInternal={true}
+                onChange={(timeZone) => {
+                  onToggleChangeTimeSettings();
+
+                  if (isString(timeZone)) {
+                    onChangeTimeZone(timeZone);
+                  }
+                }}
+                onBlur={onToggleChangeTimeSettings}
+              />
+            </section>
+          ) : (
+            <section
+              aria-label={selectors.components.TimeZonePicker.container}
+              className={cx(style.timeZoneContainer, style.timeSettingContainer)}
+            >
+              <Field className={style.fiscalYearField} label={'Fiscal year start month'}>
+                <Select
+                  value={fiscalYearStartMonth}
+                  options={[
```

Comment:
Perhaps this should be moved somewhere else? Maybe this exists elsewhere?

Reply:
Maybe let's rename [`rangeOptions.ts`](https://github.com/grafana/grafana/blob/718d6fb25d61d73614e48982c01a7c0a52c42bd8/packages/grafana-ui/src/components/TimePicker/rangeOptions.ts) to `options.ts` and export these options from there?